documentation/IDTA-01002-3/modules/ROOT/pages/http-rest-api/http-rest-api.adoc

```diff
@@ -422,27 +434,28 @@ URL-encoded IdShortPath
 |PatchSubmodelElementByPath |PATCH |/submodel/submodel-elements/\{idShortPath} a|
 use separated idShort path of this element
 
-path-suffix=$metadata/$value or no suffix for normal
+path-suffix=/$metadata or /$value
 
 URL-encoded IdShortPath
 
+For /$metadata: +
+The serialised SubmodelElement in the request body must not contain any values. +
+The server must overwrite all metadata-relevant SubmodelElement fields with the received fields. This means in particular that previously used fields of the SubmodelElement instance that are missing in the request body indicate a deletion of them. It is not possible to change single items in contained arrays. In particular, a client must therefore send the complete content of arrays (e.g. for `description` or `displayName`).
+
+For /$value: +
```

Comment:
Similar to PATCH /submodel/$value, however, for certain cases this potentially might work, e.g. if limited to only work on elements of type `Property`. Even this would still require defining additional semantics such as what happens when having a property of type DateTime and sending a numeric value from which we cannot infer the concrete type from the valueOnly serialization?

Reply:
This is a broader topic. We currently do not require the server to check whether the value `value` actually fits to the format claimed in `valueType`.